blueprint/office-365.md

```diff
@@ -118,59 +118,63 @@ Show the users name on the top navigation bar when the user is signed in | Yes |
 
 ### Role based access control
 
-Role Based Access Control (RBAC) defines what an end user or administrator can do. In relation to system administration, RBAC provides various roles each of which can only perform certain tasks. For example, help desk staff may be able to only view certain resources, whereas system administrators could view, create, and delete those resources. Office 365 provides a subset of administrative roles available in Microsoft Azure.
+Role Based Access Control (RBAC) defines what a user or administrator has access to (data or user actions) based on the user's job function (role) within the organisation. In relation to system administration, RBAC provides various roles each of which can only perform certain tasks. For example, help desk staff may be able to only view certain resources, whereas system administrators could view, create, and delete those resources. Office 365 provides a subset of administrative roles available in Microsoft Azure.
 
 Privileged Identity Management (PIM) can be leveraged to enhance the RBAC model for Azure Active Directory role-based management access, and parts of other Microsoft services like Office 365 and Microsoft Endpoint Manager. PIM requests are made through the Azure portal for elevated access only when they are required, and access is expired after a specified period.
 
+PIM requires an Azure AD Premium P2 license which is included with Microsoft 365 E5 licensing or E5 Security Step-up licensing .
+
 The following Office 365 roles can be assigned via PIM:
 
-* Exchange administrator
-* Exchange recipient administrator
-* SharePoint administrator
-* Teams administrator
-* Teams Communications administrator
-* Teams Communications support engineer
-* Teams Communications support specialist
-* Power BI Administrator
-* Power Platform administrator
-* Customer Lockbox Administrator
-* Intune administrator
-* Office Apps administrator
-* Message Center Privacy Reader
-* Message Center Reader
-* Security administrator
-* Security reader
+* Exchange administrator.
+* Exchange recipient administrator.
+* SharePoint administrator.
+* Teams administrator.
+* Teams Communications administrator.
+* Teams Communications support engineer.
+* Teams Communications support specialist.
+* Power BI Administrator.
+* Power Platform administrator.
+* Customer Lockbox Administrator.
+* Intune administrator.
+* Office Apps administrator.
+* Message Center Privacy Reader.
+* Message Center Reader.
+* Security administrator.
+* Security reader.
+
+Note, using PIM for the SharePoint administrator role, the Device administrator role, and roles trying to access the Microsoft Security and Compliance Center might experience delays of up to a few hours after activating the role, see [](https://docs.microsoft.com/en-us/azure/active-directory/privileged-identity-management/pim-roles) for further information. 
```

Comment:
Is this the right link about the delay? It sends me to a page titled: `Roles you can't manage in Privileged Identity Management`

Reply:
Yeah its correct, if you scroll down the page it is mentioned in a <Note!> block warning about the issue with the delay using PIM for various roles.